syntax = "proto3";
/**package com.fiberhome.fums.proto;**/
package proto;
option java_package = "com.fiberhome.fums.proto";   
option java_outer_classname = "Event";
import "CommonDefine.proto";

message EventData
{
	enum EVENTTYPE
	{
		AGENT_START  = 0;
		APP_START    = 1;
		APP_STOP     = 2;
		CTRLAPPRST   = 3;
        COMM_EVENT   = 4;        		
	}
	EVENTTYPE  type                 = 1;
	string     eventProbableCause   = 2;
	string     eventEntityInstance  = 3;
	EnumDefineData.AppTypeEnum appTypeCode = 4;
	string     eventOccurTime       = 5;
	string     eventMessage         = 6;
	int32      eventSeqNo           = 7;
}
